### PR TITLE
Edit Profile and Create Organization with default email

### PR DESCRIPTION
**Pull request by Lord-Chris**
Edit Profile

https://user-images.githubusercontent.com/58702861/136476555-de1213a4-6453-4eed-9d8d-8c54fa3e2641.mp4

Create Organization

https://user-images.githubusercontent.com/58702861/136476607-80247fb4-1357-4bf9-9f77-85197e689c39.mp4

### CODE REVIEW

File: lib/ui/view/edit_profile/widget/edit_profile_body.dart [hunk 1]
```diff
@@ -63,23 +68,32 @@ class Body extends ViewModelWidget<EditProfileViewModel> {
               ),
             ),
             TextFormField(
-              initialValue: '',
-              onChanged: (value) {},
+              initialValue: viewModel.userModel.displayName,
+              onChanged: (value) {
```
[the remark this answers is not in the document]
Comment (jeremiahseun) [reply lib/ui/view/edit_profile/widget/edit_profile_body.dart]: In the view, let it be like this
`onChanged: model.onChanged(**theValueYouWantToWatch**)`

File: lib/ui/view/edit_profile/widget/edit_profile_body.dart [hunk 2]
```diff
@@ -63,23 +68,32 @@ class Body extends ViewModelWidget<EditProfileViewModel> {
...
                 labelText: DisplayName,
                 helperText: DisplayNameDescription,
                 helperMaxLines: 3,
               ),
             ),
             TextFormField(
-              initialValue: '',
-              onChanged: (value) {},
+              initialValue: viewModel.userModel.bio,
+              onChanged: (value) {
```
[the remark this answers is not in the document]
Comment (jeremiahseun) [reply lib/ui/view/edit_profile/widget/edit_profile_body.dart]: This place too

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Comment by YenumJay** [comment]
Resolve conflicts

**Review by pauleke65 — APPROVED** [review]
LGTM

**Pull request merged**